rules/falco_rules.yaml

```diff
@@ -2687,26 +2747,14 @@
 - rule: Delete or rename shell history
   desc: Detect shell history deletion
   condition: >
-    (modify_shell_history or truncate_shell_history) and
-       not var_lib_docker_filepath and
-       not proc.name in (docker_binaries)
+    (modify_shell_history or truncate_shell_history) 
+    and not var_lib_docker_filepath 
+    and not proc.name in (docker_binaries)
   output: Shell history deleted or renamed (file=%fd.name name=%evt.arg.name path=%evt.arg.path oldpath=%evt.arg.oldpath evt_type=%evt.type user=%user.name user_uid=%user.uid user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty exe_flags=%evt.arg.flags %container.info)
   priority:
     WARNING
   tags: [maturity_incubating, host, container, process, filesystem, mitre_defense_evasion, T1070]
 
-# This rule is deprecated and will/should never be triggered. Keep it here for backport compatibility.
```

Comment:
Proposing to definitely remove this one as if its a duplicate it may be very confusing.